lib/react.dart

```diff
@@ -2116,207 +2116,3 @@ dynamic view = validateJsApiThenReturn(() => ReactDomComponentFactoryProxy('view
 
 /// The SVG `<vkern>` [VkernElement].
 dynamic vkern = validateJsApiThenReturn(() => ReactDomComponentFactoryProxy('vkern'));
-
-/// Create React DOM `Component`s by calling the specified [creator].
-_createDOMComponents(creator) {
-  a = creator('a');
-  abbr = creator('abbr');
-  address = creator('address');
```

Comment:
How long has this been hanging around unused?

Reply:
Looks like almost 3 years 💀 https://github.com/Workiva/react-dart/commit/1ef1196679239187f45d277de2123994785ffdfd#diff-971a483a502cde928eaed632c11bd797a2a1e6c112fb9152dbe374291d7c8dbaL2804